### PR TITLE
Add filters that can listen for multiple events

### PR DESCRIPTION
**Pull request by justinbarry**

### CODE REVIEW

File: packages/augur-sdk/src/state/db/BlockAndLogStreamListener.test.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Block } from "ethereumjs-blockstream";
 import { BlockAndLogStreamerDependencies, ExtendedLog } from "blockstream-adapters";
 import { EventLogDBRouter } from "./EventLogDBRouter";
 import { ParsedLog } from "@augurproject/types";
+import { favoriteButtonFavorite } from "@augurproject/ui/src/modules/common/buttons.styles.less";
```
Comment (nuevoalex) [inline packages/augur-sdk/src/state/db/BlockAndLogStreamListener.test.ts]: wut

**Review by nuevoalex — APPROVED** [review]
👍 

Is there a separate task for the DerivedDB event amalgamation and handling?

**Comment by bthaile** [comment]
might want to rebase, upload block file might has change from recent upload to kovan

**Review by nuevoalex — APPROVED** [review]
👍

**Pull request merged**